internal/bminventory/inventory.go

```diff
@@ -4091,6 +4112,11 @@ func (b *bareMetalInventory) uploadLogs(ctx context.Context, params installer.Up
 			log.WithError(err).Errorf("Failed update cluster %s controller_logs_collected_at flag", params.ClusterID)
 			return common.NewApiError(http.StatusInternalServerError, err)
 		}
+		err = b.clusterApi.UpdateLogsProgress(ctx, currentCluster, string(models.LogsStateCollecting))
```

Comment:
collecting is set via UploadLogs. It saves a round trip to the service from the agent and controller. Since I did not want to break existing APIs and UI behavior I did not unify the updating of collecting and the timestamp but that can be done in later stage when the feature is proven stable